LICENSE

```diff
@@ -44,7 +44,7 @@
       of this License, Derivative Works shall not include works that remain
       separable from, or merely link (or bind by name) to the interfaces of,
       the Work and Derivative Works thereof.
- 
+
```

Comment:
@mbad0la It's a space! I've got Atom set up to remove trailing whitespace, so it got rid of this automatically.

@gr2m I don't think the space is necessary for parsing this LICENSE; if you check out some of the other Hoodie LICENSE files, they are identical except don't have this extra space (check out https://github.com/hoodiehq/hoodie-client/blob/master/LICENSE#L47 for example!). Happy to put it back in if y'all prefer, but it shouldn't cause any parsing issues on your side :)